code/edge/src/androidTest/java/com/adobe/marketing/mobile/ConfigOverridesFunctionalTests.kt

```diff
@@ -238,14 +209,22 @@ class ConfigOverridesFunctionalTests {
         )
         assertEquals(1, resultRequests.size.toLong())
 
-        val payloadJSONData = getPayloadJson(resultRequests[0])
-        val eventJSONData = payloadJSONData.getJSONArray("events").getJSONObject(0)
-        val metaJSONData = payloadJSONData.getJSONObject("meta")
-
-        assertNotNull(metaJSONData.opt("konductorConfig"))
-        assertNull(metaJSONData.opt("configOverrides"))
-
-        JSONAsserts.assertExactMatch(expectedXdmDataMap, eventJSONData)
+        val expected = """
+        {
+          "events": [
+            {
+              "xdm": {
+                "testXdmKey": "testXdmValue"
+              }
+            }
+          ],
+          "meta": {
+            "konductorConfig": {}
+          }
+        }
+        """
+        // Verify that configOverrides is not present in the meta data
+        assertExactMatch(expected, getPayloadJson(resultRequests[0]), KeyMustBeAbsent("meta.configOverrides"))
```

Comment:
This would fail for konductorConfig right? 

I believe with the exactMatch, typeMatch, scope, do we really need KeyMustBeAbsent, counts?

Reply:
Okay got it. So everything is validated against expected. So if it is missing anything or actual has more stuff it might get not get checked unless we assert on size and key not present and such things?